packages/core/src/app/address/AddressSelect.tsx

```diff
@@ -90,7 +90,7 @@ const AddressSelect = ({
                 </DropdownTrigger>
             </div>
 
-            {shouldShowPayPalConnectLabel && <PoweredByPaypalConnectLabel />}
+            {shouldShowPayPalFastlaneLabel && <PoweredByPayPalFastlaneLabel />}
```

Comment:
```suggestion
            {Boolean(shouldShowPayPalFastlaneLabel) && <PoweredByPayPalFastlaneLabel />}
```

Reply:
I don't think it is necessary to use Boolean() here since shouldShowPayPalFastlaneLabel is a boolean variable
<img width="1184" alt="Screenshot 2024-03-04 at 10 41 52" src="https://github.com/bigcommerce/checkout-js/assets/25133454/505cad66-3919-49b7-a9ef-5b1956b23e0e">